oonib/bouncer/handlers.py

```diff
@@ -190,6 +190,68 @@ def filterByNetTests(self, requested_nettests):
             nettests.append(nettest)
         return {'net-tests': nettests}
 
+    def formatCollectorsWithoutPolicy(self):
+        ''' Formats the collectors without policy for the new
+            /api/v1/collectors endpoint. '''
+        results = []
+        for collector in self.knownCollectorsWithoutPolicy:
+            results.append({'type': 'onion', 'address': collector})
+            section = self.bouncerFile.get('collector')
+            if not section: continue
+            info = section.get(collector)
+            if not info: continue
+            for alt in info['collector-alternate']:
+                r = {}
+                for k in ('type', 'address', 'front'): r[k] = alt.get(k)
+                if not r['type'] or not r['address']: continue
+                if not r['front']: del r['front']
+                results.append(r)
```

Comment:
The section appears to be duplicated in the two sections, perhaps it can be abstracted as a function called something like `get_alternate_address()`?

Reply:
Done!